README.markdown

```diff
@@ -1,3 +1,4 @@
-# serverless
+# node-lambda
```

Comment:
@rszewczyk what are your thoughts on this name for the repo? `serverless` is misleading so I am searching for a more appropriate name

Reply:
what's misleading about it?